test/linear_samplers/proc_block_row_gaussian.jl

```diff
@@ -0,0 +1,29 @@
+# This file is part of RLinearAlgebra.jl
+# This file was written by Nathaniel Pritchard  
+
+module ProceduralTestLSBRGaussSampler
+
+using Test, RLinearAlgebra, Random, LinearAlgebra
+
+Random.seed!(1010)
+
+@testset "LSBR Gaussian Sampling -- Procedural" begin
+
+    # Verify appropriate super type
+    @test supertype(LinSysBlkRowGaussSampler) == LinSysBlkRowSampler
+
+    # Test construction
+    A = rand(5,10)
+    b = rand(5)
+    x = rand(10)
+
+    samp = LinSysBlkRowGaussSampler()
+
+    S, SA, res  = RLinearAlgebra.sample(samp, A, b, x, 1)
+
+    @test norm(SA - S * A) < eps() * 1e2
+    @test norm(res - (S * A * x - S * b)) < eps() * 1e2
+
```

Comment:
Need to test assertions are triggered correctly and warning statements are triggered correctly; need to test beyond iteration 1

Reply:
I have added the assertion checks and run for 5 iterations